pkg/channel/message_dispatcher.go

```diff
@@ -205,6 +220,7 @@ func (d *MessageDispatcherImpl) executeRequest(ctx context.Context, url *url.URL
 	if err != nil {
 		execInfo.Time = dispatchTime
 		execInfo.ResponseCode = nethttp.StatusInternalServerError
+		execInfo.ResponseBody = []byte(fmt.Sprintf("sender.sendWithRetries() failed: err=%s", err.Error()))
```

Comment:
Simplify this to just `dispatch error: %s`

Reply:
👍